wolox-training/jc-swift
# Project rename

## Summary ##
**Trello card**: https://trello.com/b/THE_PROJECT_BOARD/THE_CARD

A brief description of the pull request

## Test Plan ##

  * Open the app
  * Login with username 'test@gmail.com' password '123456'
  * You should see a tost with spinner while logging in.
  * The main tab bar view should appear after a successfull login.

## Screenshots ##

### iPhone 4s

Drag and drop screen shots

### iPhone 5c/5/5s

Drag and drop screen shots

### iPhone 6

Drag and drop screen shots

### iPhone 6+

Drag and drop screen shots

## Screencast ##

Drag and drop gif screen cast:
 * If you can prove the functionality using the simulator use [LICECap](http://www.cockos.com/licecap/) to record the screencast.
 * If the feature you are adding must be tested in a real device (you depend on bluetooth for example) use QuickTime to record the screencast. Check [this](http://www.tekrevue.com/tip/record-iphone-screen-quicktime/) tutorial on how to record a screencast with Quicktime and then check [this](https://gist.github.com/dergachev/4627207) tutorial on how to convert the `.mov` to `.gif` file in order to be able to attach it to the pull request.

## Known issues ##

If there are known issues or missing functionality report them
